lib/main.dart

```diff
@@ -15,6 +17,16 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       theme: AppTheme.light,
       home: LoginPage(),
+      localizationsDelegates: [
+        AppLocalizations.delegate,
+        GlobalMaterialLocalizations.delegate,
+        GlobalWidgetsLocalizations.delegate,
+        GlobalCupertinoLocalizations.delegate,
+      ],
+      supportedLocales: [
+        Locale('en', ''),
+        Locale('vi', ''),
```

Comment:
To avoid hardcoding support locales, I suggest to use : 
AppLocalizations.delegate.supportedLocales instead

Reply:
that's good to know 🙌

Reply:
@sleepylee resolved 👌